bio.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" style="background-color:rgb(48, 48, 48); color: rgb(226, 225, 223);">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Bio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>Biography</h1>
+        <p>Junior Engineer</p>
+
```

Comment:
Since your nav tag is part of the header, it should generally be placed inside there.  This would also be the same on all pages.